malaria24/settings/docker.py

```diff
@@ -0,0 +1,31 @@
+"""Django settings for use within the docker container."""
+
+from os import environ
+from os.path import abspath, dirname, join
+
+import dj_database_url
+
+from .production import *  # noqa: F401, F403
+
+
+SECRET_KEY = environ.get('DOCKER_SECRET_KEY') or DEFAULT_SECRET_KEY
```

Comment:
Is there a reason this is called `DOCKER_SECRET_KEY`? Not sure if any of our other apps do this. Also if someone accidentally sets the `SECRET_KEY` env var when using Docker, they'll just end up with the default key.

Reply:
yeah, I change it back to the secret key